notebooks/growth_model_classification.ipynb

```diff
@@ -0,0 +1,1053 @@
+{
```

Comment:
<p>reword to: <code>....we successfully analyzed different customer accounts based on their tenure and monthly revenue, resulting in the following accomplishments</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>